db/schema.rb

```diff
@@ -1765,7 +1765,7 @@
     t.datetime "updated_at"
     t.string "name"
     t.string "transfer_class_name"
-    t.string "transfers", limit: 1024
+    t.string "transfers", limit: 10240
```

Comment:
is this intentional? looks like a typo

Reply:
Yes, transfers is a string that contains a JSON with all transfers. 1024 didnt have enough size to fit the definition of the transfers for 384 wells, so I had to migrate the column